app/(platformMap)/layout.tsx

```diff
@@ -48,7 +54,7 @@ export default function Layout({
         </Col>
       </Row>
 
-      {(isPlatformView && bottom) ?? <Row>{bottom}</Row>}
+      {(isPlatformView && isClient && bottom) ?? <Row>{bottom}</Row>}
```

Comment:
Why do we only need to check for client for the bottom content?

Reply:
Can we push client only rendering down to the chart components themselves?

Reply:
I've got [`'use client'` on at least some of the charts](https://github.com/gulfofmaine/Neracoos-1-Buoy-App/blob/main/src/components/Charts/LargeTimeSeries.tsx#L1), so it may be something in between.

It looks like there may be `async` missing on some of the `page.tsx` at this level https://github.com/gulfofmaine/Neracoos-1-Buoy-App/blob/30dbb04393f6359314ec057c77fc8e409f6c5f69/app/(platformMap)/%40bottom/platform/%5BplatformId%5D/observations/%5BstandardName%5D/page.tsx#L5